README.md

```diff
@@ -14,7 +14,7 @@ scheduled weekly, recurring events.
 The intended clientele is Narcotics Anonymous Service bodies. They implement a BMLT
```

Comment:
clientele?

Reply:
user?

Reply:
The intended users our Narcotics Anonymous Service bodies.